wdioJasmineHelper.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path');
+
+const DEFAULT_OPTIONS = {
+    configFile: path.resolve(__dirname, 'babel.config.js'),
+    root: __dirname,
+    rootMode: 'root',
+    ignore: [],
+    extensions: ['.js']
+};
+
+require('@babel/register')(DEFAULT_OPTIONS);
+
+module.exports = function _jasminRegister() {
```

Comment:
I think there may be a typo missing an 'e', should this function rather be called _jasmineRegister()?

Reply:
Good catch @nlealffdc. TBH I copied this over from the official utam-js-recipes example repo:
https://github.com/salesforce/utam-js-recipes/blob/main/wdioJasmineHelper.js#L20

@olivier-martin-sf any clue on this possible typo?

Reply:
This is indeed a typo, however, the function could be called any name or even be anonymous. I don't think that affects the runtime behavior at all but it would be more elegant to fix the typo for sure!

Reply:
Thanks for chiming in. I'll fix it in both repos.